azure_batch/main.tf

```
@@ -0,0 +1,183 @@
+/*
+
+# Purpose
+
+Create an Azure Batch account [1] with a pool populated of Docker-compatible compute nodes [2].
```

Suggestion to expand on what the module does (tell me if it's not accurate):
```suggestion
Create an Azure Batch account [1] with an autoscaling pool of compute nodes that run a Docker
image [2] and can read resource files from a storage account and write the output back to that
storage account.
```